Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean

```diff
@@ -164,6 +164,9 @@ theorem pi_pos : 0 < π :=
   lt_of_lt_of_le (by norm_num) two_le_pi
 #align real.pi_pos Real.pi_pos
 
+theorem pi_nonneg : 0 ≤ π :=
+  pi_pos.le
+
 theorem pi_ne_zero : π ≠ 0 :=
   ne_of_gt pi_pos
```

Comment:
```suggestion
   pi_pos.ne'
```
Does it work?

Reply:
Oops, your suggested change had three spaces instead of one. :/

Reply:
Sorry about that!  I was on mobile...